software/services/employee/app/models/user.py

```diff
@@ -1,7 +1,8 @@
 from pydantic import BaseModel
-from typing import Dict
+from models import cv
 
 class User(BaseModel):
     username : str
     email : str
     password : str
+    user_cv : cv
```

Comment:
Co do aktualizacji CV, lepiej też zrobić oddzielnie aktualizację CV i oddzielnie użytkownika. Mówię tak, bo to sa oddzielne tabele w bazie danych a też nie wiemy czy się nie bedą zmieniać.

Ale w twoim module mogą być połączone i możesz przekazywać sobie te modele, a korzystać tylko z tego, co potrzeba w danym momencie.